test/fixtures/output/flink/shell/shell.golden

```diff
@@ -0,0 +1,232 @@
+
+TEXT OUTPUT 
+========================================================================== 
+
+Welcome! 
+To exit, press Ctrl-Q or type "exit". 
+
+[Ctrl-Q] Quit [Ctrl-S] Toggle Smart Completion 
+ 
+Smart Completion is now disabled                                                                    
+> > > use catalog default;> use catalog default;> use catalog default;
```

Comment:
Looks like we're submitting multiple statements in parallel, without waiting for the previous one to execute?

Reply:
No I think this just comes from @Guttz removing the ansi sequences to make the golden file more readable

Reply:
Any way we can clean up the formatting here?

Reply:
I'll disable the colors and then there should be no need for removing the ansi sequences, which will in turn make the golden file more beautiful to look at 😄